docs/application/native/guides/text-input/tts.md

```diff
@@ -519,23 +529,22 @@ To set and get the options about the TTS engine:
   ```
 
 <a name="text"></a>
-## Adding Text
+## Add text
 
 There are defined values, `TTS_VOICE_TYPE_AUTO` and `TTS_VOICE_SPEED_AUTO` for following the default TTS setting. Those values, and the minimum and maximum limits for the speed, are defined in the `tts.h` header file.
```

Comment:
A slight grammatical error was fixed.
```suggestion
There are defined values, `TTS_VOICE_TYPE_AUTO` and `TTS_VOICE_SPEED_AUTO`, for following the default TTS setting. Those values, and the minimum and maximum limits for the speed, are defined in the `tts.h` header file.
```

Reply:
@stom-hwang,
We added a comma at the end of `TTS_VOICE_SPEED_AUTO`, perhaps you missed it on your last commit.